UPDATE.md

```diff
@@ -65,6 +67,10 @@ Beta 2, then from Beta 2 to Beta 3, in that order.
   actively using Page Manager, you must add it to your project as an explicit
   dependency in order to continue to using it. Otherwise, you should uninstall
   it before updating to the next version of Lightning.
+* Lightning now supports exposing all Drupal entities as JSON, in the standard
+  JSON API format. To enable this feature, install the OpenAPI, OpenAPI Redoc
+  Documentation, and JSON API modules. Then install the Content API module from
+  the Lightning package.
```

Comment:
I understand why it was set up this way - it makes sense and it will save us from dependency problems/complaints later on.

Still, I feel like people will want to just turn this on to experiment. And this is a barrier.

Could we add a subcomponent of Content API that does what Content API currently does and make that and the contrib modules a dependency of Content API? That way, there's a single module that turnkey adds the functionality - but no direct dependencies between the functional modules.